core/src/main/java/org/apache/accumulo/core/conf/Property.java

```diff
@@ -1412,6 +1412,15 @@ public enum Property {
           + "constraint.",
       "2.0.0"),
 
+  TABLE_ENABLE_ERASURE_CODES("table.file.ec.enable", "false", PropertyType.BOOLEAN,
+      "This determines if Accumulo will manage erasure codes on a table."
+          + " When setting this to true must also set erasure.code.policy.",
+      "2.1.4"),
+
+  TABLE_ERASURE_CODE_POLICY("table.file.ec.policy", "", PropertyType.STRING,
```

Comment:
Could call `DistributedFileSystem.getAllErasureCodingPolicies()` when this is set to validate that the policy exists and is enabled. Would likely need to check all volumes.